homeassistant/components/konnected/strings.json

```diff
@@ -18,11 +18,11 @@
       }
     },
     "error": {
-      "cannot_connect": "Unable to connect to a Konnected Panel at {host}:{port}"
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
```

Comment:
We may need to update the config flow to not pass the host and port template variables, or revert the change.

Reply:
Seems it should work anyway without further changes.